homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/about.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
Please remove this page back to `index.html` - you have to have one base starting point in your project

Reply:
still seeing this 
<img width="1207" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/24bd8ebb-533c-43ae-baf1-dbc02ef9edae">

Reply:
I hope it has changed now :)